test/cpp/jit/test_gpu.cpp

```diff
@@ -9275,6 +9275,51 @@ TEST(NVFuserTest, Issue329_CUDA) {
   TORCH_CHECK(at_t3.allclose(outputs[1]));
 }
 
+TEST(NVFuserTest, FusionIssue382_CUDA) {
+  Fusion fusion;
+  FusionGuard fg(&fusion);
+
+  auto tv0 = makeSymbolicTensor(2);
+  fusion.addInput(tv0);
+
+  auto tv1 = add(tv0, new Float(1));
+  auto tv2 = broadcast(tv1, {false, false, true});
+  auto tv3 = makeSymbolicTensor(3);
+  fusion.addInput(tv3);
+  auto tv4 = add(tv2, tv3);
+  fusion.addOutput(tv4);
+
+  tv2->merge(1);
```

Comment:
Does this statement do anything?

Reply:
Yes. The bug of issue #382 is caused by this merge.